src/pages/WSLIntegrations.vue

```diff
@@ -45,8 +45,15 @@ export default Vue.extend({
 
 <template>
   <wsl-integration
```

Comment:
The page looks weird because the description doesn't scroll with the body:
![image](https://user-images.githubusercontent.com/3977982/178854583-ae1675fe-2458-4cb8-b046-9f4ab404380c.png)
But I think it's okay to ignore it here, because we'll be getting rid of it shortly anyway.

Reply:
I'm not going to worry this one 🙂